src/components/features/decoupled-components/hero-banner-component/hero-banner-component.module.scss

```diff
@@ -0,0 +1,5 @@
+$secondary-color: red;
+
+.custom_button_class button {
```

Comment:
Maybe rename it so it is less generic? This class applies to the teaser component, so maybe use .teaser button? Then 'teaser' can indicate the context for the 'button', so it can be styled in a specific way.